conda_build/build.py

```diff
@@ -36,6 +36,10 @@
 
 
 on_win = (sys.platform == 'win32')
+if sys.platform.startswith('openbsd'):
```

Comment:
@ilanschnell, my only suggestion is to change this line to:

```python
if 'bsd' in sys.platform:
```

so that this PR applies to the other BSD's (i.e. FreeBSD and NetBSD).